unidist/core/backends/mpi/core/communication.py

```diff
@@ -156,7 +156,7 @@ def get_topology():
 # ---------------------------- #
 
 
-def mpi_send_object(comm, data, dest_rank):
+def mpi_send_object(comm, data, dest_rank, tag=0):
```

Comment:
```suggestion
def mpi_send_object(comm, data, dest_rank, tag=common.MPITag.OBJECT):
```

Reply:
I feel it might be good to have default tag as OBJECT for this method, and have an extra method mpi_send_operation for sending operation.

Reply:
makes sense, fixed